TestWeightDevice test case failed

### Ⅰ. Issue Description
```
--- FAIL: TestWeightDevice_Type (0.00s)
    --- FAIL: TestWeightDevice_Type/implement_WeightDevice_as_pflag.Value_interface (0.00s)
    	blkio_test.go:239: WeightDevice.Type() = strings, want value
```

```
--- FAIL: TestThrottleBpsDevice_Type (0.00s)
    --- FAIL: TestThrottleBpsDevice_Type/implement_ThrottleBpsDevice_as_pflag.Value_interface (0.00s)
    	blkio_test.go:505: ThrottleBpsDevice.Type() = strings, want value
```

```
--- FAIL: TestThrottleIOpsDevice_Type (0.00s)
    --- FAIL: TestThrottleIOpsDevice_Type/implement_ThrottleIOpsDevice_as_pflag.Value_interface (0.00s)
    	blkio_test.go:791: ThrottleIOpsDevice.Type() = strings, want value

```

if we change `blkioOptsType`,  we also should change the test case 

### Ⅱ. Describe what happened


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/apis/opts/config/blkio_test.go
+++ b/apis/opts/config/blkio_test.go
@@ -227,7 +227,7 @@ func TestWeightDevice_Type(t *testing.T) {
 					},
 				},
 			},
-			want: "value",
+			want: "strings",
 		},
 	}
 	for _, tt := range tests {
@@ -493,7 +493,7 @@ func TestThrottleBpsDevice_Type(t *testing.T) {
 					},
 				},
 			},
-			want: "value",
+			want: "strings",
 		},
 	}
 	for _, tt := range tests {
@@ -779,7 +779,7 @@ func TestThrottleIOpsDevice_Type(t *testing.T) {
 					},
 				},
 			},
-			want: "value",
+			want: "strings",
 		},
 	}
 	for _, tt := range tests {

